spec/takeaway_spec.rb

```diff
@@ -0,0 +1,69 @@
+require 'takeaway'
+
+describe Takeaway do
+  let(:menu) { double :menu }
+  let(:test_menu) { {'Pizza' => 7} }
+  subject(:takeaway) { described_class.new (twilio_class)}
+  let(:twilio_class) {double :twilio_class, :new => client }
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)